### PR TITLE
test-bench: section on installing testing deps

### PR DESCRIPTION
**Pull request by deeplow**

### CODE REVIEW

**Review by andrewdavidwong — CHANGES_REQUESTED** [review]
Generally, the names of things should be written the same way they appear in their respective doc pages or their official names outside of Qubes, e.g., `Split GPG`, `Salt`, and `SaltStack`. Please avoid this all-lowercase-with-no-spaces naming (unless that's actually how the thing is named).

**Review by unman — COMMENTED** [review]
You should fix the numerous other spelling/grammar errors on this page at the same time.

File: developer/debugging/test-bench.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ The following commands should work for you, but do keep the provisioning scripts
    sudo cp -a system-tests/ /srv/salt/
    sudo qubesctl top.enable system-tests
 
-   # install openqa configuration
+   # install them same configuration as the one in OpenQA
```
Comment (unman) [inline developer/debugging/test-bench.md:123]: **the same**

Comment (deeplow) [reply developer/debugging/test-bench.md]: thanks for catching it

**Comment by deeplow** [comment]
> Generally, the names of things should be written the same way they appear in their respective doc pages or their official names outside of Qubes, e.g., `Split GPG`, `Salt`, and `SaltStack`. Please avoid this all-lowercase-with-no-spaces naming (unless that's actually how the thing is named).

Done.

**Review by andrewdavidwong — APPROVED** [review]

**Comment by andrewdavidwong** [comment]
Actually, @unman, would you be willing to review this so we can give Marek a break?

**Comment by unman** [comment]
On Sun, Jul 18, 2021 at 06:52:12PM -0700, Andrew David Wong wrote:
> Actually, @unman, would you be willing to review this so we can give Marek a break?
> 

Yes, tomorrow.

File: developer/debugging/test-bench.md [hunk 2]
```diff
@@ -99,6 +99,43 @@ Internet access is intentionally disabled by default in dom0. But to ease the de
...
+
+   # Install the same configuration as the one in openQA
+   QUBES_VERSION=4.1
+   PILLAR_DIR=/srv/pillar/base/update
+   sudo mkdir -p $PILLAR_DIR
+   printf 'update:\n  qubes_ver: '$QUBES_VERSION'\n' | sudo tee $PILLAR_DIR/init.sls
+   printf "base:\n  '*':\n    - update\n" | sudo tee $PILLAR_DIR/init.top
+   sudo qubesctl top.enable update pillar=True
+
+   sudo systemctl restart qubesd
```
Comment (marmarek) [inline developer/debugging/test-bench.md:131]: qubesd restart is not needed after just adding salt recipes.

**Review by marmarek — CHANGES_REQUESTED** [review]
Besides one minor note, looks good.

**Comment by andrewdavidwong** [comment]
Ok, looks like @marmarek will be reviewing this instead of @unman after all.

> Besides one minor note, looks good.

Shall I wait for your usual signed approval comment, per our [policy](https://www.qubes-os.org/doc/how-to-edit-the-documentation/#security)?

**Pull request merged**